cypress/support/fragments/orders/orderLines.js

```diff
@@ -129,8 +129,11 @@ export default {
       Select({ name: 'physical.materialType' }).choose('book'),
       Button('Add location').click(),
       Button('Location look-up').click(),
-      Select({ name: 'institutionId' }).choose('Københavns Universitet'),
+      // Someone create new institution, and on ui sometimes we need to select Institution.
+      // Select({ name: 'institutionId' }).choose('Københavns Universitet'),
       Select({ name: 'campusId' }).choose('Online'),
```

Comment:
If no one will create institutions and locations it will all be without choice and unnecessary actions.But someone creates them, do not clean up in time and I need to change the configuration in this test all the time .

Reply:
can you create special institution and use it into your test. And remove it at the end of your test. Off cause,  I mean api calls.